src/common_fits.jl

```diff
@@ -206,7 +206,7 @@ function analyzeRB(ypts, seqlengths)
 
 	numRepeats = length(ypts)/length(seqlengths);
 
-	xpts = seqlengths[1 + floor((0:length(ypts)-1)./numRepeats)];
+  	xpts = seqlengths[1 + floor(Int64, (0:length(ypts)-1) .÷ numRepeats)];
```

Comment:
`julia> using Qlab`
`julia> DATA_PATH = "/Volumes/share/Matt/RGST/170404RGST/";`
`julia> _, data = load_data(DATA_PATH * "1_100_RB.h5");`
`julia> ypts = real(data["data"]);`
`julia> seqLength = [2,4,16,32,64,128,256];`
`julia> Qlab.analyzeRB(ypts,seqLength)`
`WARNING: floor{T <: Real}(x::AbstractArray{T}) is deprecated, use floor.(x) instead.
Stacktrace:
 [1] depwarn(::String, ::Symbol) at ./deprecated.jl:64
 [2] floor(::Array{Float64,1}) at ./deprecated.jl:51
 [3] analyzeRB(::Array{Float64,1}, ::Array{Int64,1}) at /Users/mware/.julia/v0.6/Qlab/src/common_fits.jl:210
 [4] eval(::Module, ::Any) at ./boot.jl:235
 [5] eval_user_input(::Any, ::Base.REPL.REPLBackend) at ./REPL.jl:66
 [6] macro expansion at ./REPL.jl:97 [inlined]
 [7] (::Base.REPL.##1#2{Base.REPL.REPLBackend})() at ./event.jl:73
while loading no file, in expression starting on line 0
ERROR: ArgumentError: invalid index: 1.0
Stacktrace:
 [1] macro expansion at ./multidimensional.jl:474 [inlined]
 [2] macro expansion at ./cartesian.jl:62 [inlined]
 [3] macro expansion at ./multidimensional.jl:471 [inlined]
 [4] _unsafe_getindex! at ./multidimensional.jl:466 [inlined]
 [5] macro expansion at ./multidimensional.jl:459 [inlined]
 [6] _unsafe_getindex(::IndexLinear, ::Array{Int64,1}, ::Array{Float64,1}) at ./multidimensional.jl:453
 [7] macro expansion at ./multidimensional.jl:442 [inlined]
 [8] _getindex at ./multidimensional.jl:438 [inlined]
 [9] getindex at ./abstractarray.jl:875 [inlined]
 [10] analyzeRB(::Array{Float64,1}, ::Array{Int64,1}) at /Users/mware/.julia/v0.6/Qlab/src/common_fits.jl:210`

`julia> versioninfo()
Julia Version 0.6.0-pre.alpha.278
Commit 1255e82 (2017-03-28 14:29 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, haswell)`